webtau-http-groovy/src/test/groovy/com/twosigma/webtau/http/HttpGroovyTest.groovy

```diff
@@ -239,6 +240,41 @@ class HttpGroovyTest implements HttpConfiguration {
         assert varArgQuery == mapBasedQuery
     }
 
+    @Test
+    void "default user agent"() {
+        http.get('/echo-header') {
+            body['User-Agent'].should == ~/^webtau\//
+        }
+    }
+
+    @Test
+    void "custom user agent"() {
+        try {
+            cfg.userAgentConfigValue.set("test", "custom")
+
+            http.get('/echo-header') {
+                body['User-Agent'].should == ~/^custom \(webtau\/.*\)$/
+            }
+        } finally {
+            cfg.userAgentConfigValue.reset()
+        }
+    }
+
+    @Test
+    void "custom user agent without webtau and its version"() {
+        try {
+            cfg.userAgentConfigValue.set("test", "custom")
+            cfg.removeWebtauFromUserAgent.set("true", true)
```

Comment:
first parameter value is the source, not sure what "true" means in this case

Reply:
Oops, meant to say `test` like the line above.